src/Sylius/Bundle/TaxonomyBundle/Resources/config/driver/doctrine/mongodb-odm.xml

```diff
@@ -11,38 +11,26 @@
 
 -->
 
-<container xmlns="http://symfony.com/schema/dic/services"
-           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-           xsi:schemaLocation="http://symfony.com/schema/dic/services
-                               http://symfony.com/schema/dic/services/services-1.0.xsd">
-
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <parameters>
         <parameter key="sylius.repository.taxon.class">Sylius\Bundle\ResourceBundle\Doctrine\ODM\MongoDB\DocumentRepository</parameter>
     </parameters>
 
     <services>
-        <service id="sylius.doctrine.odm.mongodb.unitOfWork"
-                factory-service="doctrine.odm.mongodb.document_manager"
-                factory-method="getUnitOfWork"
-                class="Doctrine\ODM\MongoDB\UnitOfWork"
-                public="false"
-        >
+        <service id="sylius.doctrine.odm.mongodb.unitOfWork" class="Doctrine\ODM\MongoDB\UnitOfWork" public="false">
+            <factory service="doctrine.odm.mongodb.document_manager" method="getUnitOfWork" />
         </service>
 
         <service id="sylius.manager.taxon" alias="doctrine.odm.mongodb.document_manager" />
-        <service id="sylius.doctrine.odm.mongodb.metadata.taxon"
-                factory-service="sylius.manager.taxon"
-                factory-method="getClassMetadata"
-                class="Doctrine\ODM\MongoDB\Mapping\ClassMetadata"
-                public="false"
-        >
+        <service id="sylius.doctrine.odm.mongodb.metadata.taxon" class="Doctrine\ODM\MongoDB\Mapping\ClassMetadata" public="false">
             <argument>%sylius.model.taxon.class%</argument>
+            <factory service="sylius.manager.taxon" method="getClassMetadata" />
         </service>
-        <service id="sylius.repository.taxon" class="%sylius.repository.taxon.class%">
+
+        <service id="sylius.repository.taxon" class="Sylius\Bundle\TaxonomyBundle\Doctrine\ORM\TaxonRepository">
```

Comment:
Not really, I guess. At least as far as I can notice, this file should declare ODM repository instead of ORM. 🎱